prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -99,13 +72,21 @@ def __describe_instances__(self, regional_client):
                                     image_id=instance["ImageId"],
                                     launch_time=instance["LaunchTime"],
                                     private_dns=instance["PrivateDnsName"],
```

Comment:
I'd say to add a `get()` to the `Instance` attributes in this function.

Reply:
We don't know if putting a None in the attribute will break something.